test/crypto/hashsignature.js

```diff
@@ -44,4 +44,56 @@ describe('HashSigner', function() {
       expect(isVerified).to.be.false;
     });
   });
+  describe('verifyDataSignatureWithPubKey', function() {
+    var privateKey = new PrivateKey();
```

Comment:
Move vars initialization to `beforeEach` to ensure test isolation.

Reply:
I did it this way because I need them to be exactly the same for each run. I guess I can store the private key somewhere, or move it to the `before` instead of `beforeEach`